sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -566,8 +566,8 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    *
    * You can set the following CSV-specific options to deal with CSV files:
    * <ul>
-   * <li>`sep` (default `,`): sets a single character as a separator for each
-   * field and value.</li>
+   * <li>`sep` (default `,`): sets a separator for each field and value. This separator can be one
+   * or more characters. Special characters in the separator need to be escaped by a backslash.</li>
```

Comment:
I'd remove the reference to escaping here, per earlier conversation.

Reply:
Done